ddprof-exporter/src/lib.rs

```diff
@@ -31,12 +32,14 @@ pub struct Exporter {
 }
 
 /// All tags are optional. Here are the currently known tag names:
-/// * service
-/// * language
 /// * env
-/// * version
 /// * host - the agent will overwrite this; only useful in practice if the
 ///          experimental agent-less mode is being used.
+/// * language
+/// * profiler_version
+/// * runtime-id
+/// * service
+/// * version
 /// Other tags may be used, such as the ones specified by the user in the
 /// `DD_TAGS` env.
```

Comment:
Minor: It's not often that I advocate removing comments, but to be honest, by itself this description is so bare that I don't think it ends up being very helpful. So... I would just suggest removing it entirely. (The other alternative would be to improve the documentation here, but it also doesn't seem the right place to do it)